apps/web/app/(use-page-wrapper)/(main-nav)/teams/client-page.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { ShellMainAppDir } from "app/(use-page-wrapper)/(main-nav)/ShellMainAppDir";
+import { useSession } from "next-auth/react";
+import { useRouter, useSearchParams } from "next/navigation";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+
+import TeamsView, { TeamsCTA } from "~/teams/teams-view";
+
+const ClientPage = () => {
+  const searchParams = useSearchParams();
+  const router = useRouter();
+  const session = useSession();
+  const { t } = useLocale();
+  const token = searchParams?.get("token");
```

Comment:
useSearchParams() already handles the case where there are multiple values with the same name by returning the first value, so we don't need the Array.isArray check (as we did in RSC)